README.md

```diff
@@ -18,25 +18,26 @@ git clone https://github.com/corresponsabilidad-mx/backend.git
 ```
  2. Compilar con Maven
 ```
- mvn clean install
+ mvn clean package
 ```
 3. Ejecutar el fat-jar
 ```
 java -jar blog-0.0.1-SNAPSHOT.jar
 ```
-La URL de la app es: http://localhost:8080
+La URL de la app es: http://localhost:8080/blog
 
-## Endpoints
+## Ejecucion de tests
 
-Devuelve un Hola Mundo:
 ```
-GET http://localhost:8080/
+ mvn test
 ```
 
-Ingresando el parámetro name devuelve Hola + `name`:
+## Endpoints
 
+Devuelve una Lista de publicaciones:
 ```
-GET http://localhost:8080/?name=
+GET http://localhost:8080/blog/posts
 ```
 
 
+
```

Comment:
@ametrey | Sugiero que agreguemos un VERSION.md para ir añadiendo las nuevas features del API y en el README, mantengamos descripciones y guías acerca del proyecto (Test - Deploy).

Ejemplo:

## 1.0
- Proof of concept

Reply:
@julissarparco , que te parece si esto lo creamos como un Issue aparte asi no se hace de chicle este PR?

Reply:
Sí, de acuerdo 👍🏼

Reply:
agregué el archivo VERSION.md en blanco, no tengo ni idea que se pone ahí, en cuanto pueda me busco algún ejemplo